dts_test_project/customers/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please send this in a separate patch unless this is required for parallel migrations. Same for all migration files.